pyspark/test/bigdl/test_utils.py

```diff
@@ -299,11 +299,15 @@ def _do_modelTestSingleLayer(self,
                        rtol=rtol,
                        atol=atol)
 
+    # Set Keras Backend without affecting the current dim_ordering
     def __set_keras_backend(self, backend):
         if K.backend() != backend:
+            current_dim = K.image_dim_ordering()
             os.environ['KERAS_BACKEND'] = backend
             from six.moves import reload_module
             reload_module(K)
+            K.set_image_dim_ordering(current_dim)
             assert K.backend() == backend
+            assert K.image_dim_ordering() == current_dim
         if backend == "theano":
```

Comment:
why we need this? how about remove those checking?

Reply:
This is to make theano backend work with python 3.5 and 3.6